oppia/oppia-android
# Fix #1017: Copy assets script added

## Explanation
We needed a way to remove all dummy assets from domain layer temporarily and replace it with original oppia assets , so that we can build a signed apk with those original assets. Once that is done we need to revert those changes and reset everything to original, meaning replacing original oppia assets with dummy oppia assets, so that they do not get pushed on github.

## Testing approach

1. You should have a folder in local machine (will be known as `local_path_to_oppia_assets` in below explanation and codebase) which has some files in it. This folder should not be a part of Oppia-Android codebase.
2. Go to root folder of Oppia-Android codebase in terminal.
3. Run `bash scripts/copy_oppia_assets.sh <local_path_to_oppia_assets>` in terminal
4. Notice all the changes that happen to codebase like `.gitignore` file getting changed, one folder will be created just outside `oppia-android` folder which contains all dummy-assets from domain layer and all domain layer assets are replaced with files present inside `local_path_to_oppia_assets`.
5. Run `bash scripts/copy_dummy_assets.sh`, you will notice that everything changes to normal and gets reverted back to original codebase.

## Checklist
